pom.xml

```diff
@@ -26,11 +26,8 @@
         <skipUnitTests>false</skipUnitTests>
     
         <jakartaee-api.version>8.0.0</jakartaee-api.version>
-        <!-- Make this correspond to the version used in Payara: 5.201 uses (patched) 2.3.14 -->
-        <mojarra.version>2.3.14</mojarra.version>
```

Comment:
Does the version of Mojarra no longer matter?

Reply:
It does, but it is already included in the BOM. It should be kept the same as the appserver anyway, so that updating is now obsolete.

Instead, we change the appserver version in `payara.version`.